src/contour/Config.cpp

```diff
@@ -1115,7 +1115,11 @@ TerminalProfile loadTerminalProfile(UsedKeys& _usedKeys,
     profile.fonts.boldItalic.slant = text::font_slant::italic;
     softLoadFont(_usedKeys, fontBasePath, _doc["profiles"][_name]["font"], "bold_italic", profile.fonts.boldItalic);
 
+#if _WIN32
+    profile.fonts.emoji.familyName = "Segoe UI Emoji";
```

Comment:
I rember that name, yeah. And did you succeed seeing colored emoji now?

Reply:
no i'm still seeing black and white, which confuses me. Could it be the OpenGL windows used is different?

Reply:
This is resolved now, right? I mean, then we should click on resolve here, I think. :)